watertap/core/zero_order_costing.py

```diff
@@ -440,6 +440,71 @@ def cost_chlorination(blk):
         blk.config.flowsheet_costing_block.cost_flow(
             chem_flow_mass, "chlorine")
 
+    def cost_well_field(blk):
+        """
+        General method for costing well fields. Capital cost is based on well field 
+        cosntruction and pipe costs.
+        """
+
+        t0 = blk.flowsheet().time.first()
+
+        # Get parameter dict from database
+        parameter_dict = \
+            blk.unit_model.config.database.get_unit_operation_parameters(
+                blk.unit_model._tech_type,
+                subtype=blk.unit_model.config.process_subtype)
+
+        # Get costing parameter sub-block for this technology
+        A, B, pipe_cost_basis = _get_tech_parameters(
+            blk,
+            parameter_dict,
+            blk.unit_model.config.process_subtype,
+            ["capital_a_parameter", "capital_b_parameter", "pipe_cost_basis"])
+
+        # Determine if a costing factor is required
+        factor = parameter_dict["capital_cost"]["cost_factor"]
+
+        # Add cost variable and constraint
+        blk.capital_cost = pyo.Var(
+            initialize=1,
+            units=blk.config.flowsheet_costing_block.base_currency,
+            bounds=(0, None),
+            doc="Capital cost of unit operation")
+        blk.pipe_cost = pyo.Var(
+            initialize=100,
+            units=pyo.units.USD_2018,
+            bounds=(0, None),
+            doc="Piping cost for well field")
+        blk.pipe_cost_basis = pyo.Var(
```

Comment:
I saw that but I also saw it was in the `cost_chlorination()` method so I included it. I will remove it.

Reply:
@andrewlee94 Actually I am not seeing how `blk.capital_cost = Var()` can be excluded from `cost_well_field()`. Just like in `cost_chlorination()` the capex formula is a different form than that in `_general_power_law_form()` so that method is never called and the capital_cost variable is never created. Unless I am missing something?

Reply:
Ah ok ok I see. You are correct. Didn't realize `_get_tech_parameters()` returned them as `Var()`s